app/controllers/events/view/speakers/create.js

```diff
@@ -26,7 +29,9 @@ export default class extends Controller {
       this.transitionToRoute('events.view.speakers', this.model.event.id);
     } catch (e) {
       console.error('Error while saving session', e);
-      this.notify.error(this.l10n.t('Oops something went wrong. Please try again'));
+      this.errorHandler.genericError(e);
+    } finally {
```

Comment:
this line is not basically, related to this issue. This is added because the speaker form kept loading, as 'isLoading' never set to false